### PR TITLE
Emerald ore in Mountains biome

### PR DESCRIPTION
**Pull request by IceCruelStuff**
## Introduction

Adds emerald ore in mountains biome.

### CODE REVIEW

File: src/world/biome/MountainsBiome.php [hunk 1]
```diff
@@ -38,9 +41,13 @@ public function __construct(){
 		$tallGrass = new TallGrass();
 		$tallGrass->setBaseAmount(1);
 
-		$this->addPopulator($tallGrass);
+		$ores = new Ore();
+		$ores->setOreTypes([
+			new OreType(VanillaBlocks::EMERALD_ORE(), 11, 1, 0, 32)
+		]);
```
Comment (dktapps) [inline src/world/biome/MountainsBiome.php:47]: please move these lines below the tallGrass line

**Comment by dktapps** [comment]
This looks OK now. Can I ask where you got 11 for the cluster size?

There's a minor problem here because this PR can't be tested because of [this bug](https://github.com/pmmp/pocketmine-mp/blob/5a56f68991eaf6781c129174e3cf057f813249ce/src/world/generator/normal/Normal.php#L99). I'd like to see the results of this PR because I don't understand why @shoghicp didn't add this directly if it's this simple.

**Comment by IceCruelStuff** [comment]
Isn't 11 for cluster count?
https://github.com/pmmp/PocketMine-MP/blob/master/src/world/generator/object/OreType.php#L42

**Comment by IceCruelStuff** [comment]
I got all the values here
https://minecraft.gamepedia.com/Emerald_Ore#Natural_generation

**Comment by dktapps** [comment]
Yeah, I meant count, my bad. (Some part of my brain insists on flipping words that it shouldn't.)

**Comment by IceCruelStuff** [comment]
For the cluster count, I got it from the [Minecraft wiki](https://minecraft.gamepedia.com/Emerald_Ore#Natural_generation). It says emerald ore will attempt to generate 11 times per chunk.

**Pull request merged**